payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/SectionUI.kt

```diff
@@ -37,7 +37,11 @@ internal object CardStyle {
         @ReadOnlyComposable
         get() = PaymentsTheme.colors.colorComponentDivider
 
-    val cardBorderWidth: Dp = 1.dp
+    val cardBorderWidth: Dp
+        @Composable
+        @ReadOnlyComposable
+        get() = PaymentsTheme.shapes.borderStrokeWidth
```

Comment:
Out of curiosity, what is the benefit of having this as a `get`?

Reply:
We can use these annotations to enforce that this value is only fetched in a composable function. It follows the pattern of `MaterialTheme`